### PR TITLE
Docs ab

### PR DESCRIPTION
**Pull request by abax17**
Finished docs page and help me button

### CODE REVIEW

File: database/management/commands/load_fixtures.py [hunk 1]
```diff
@@ -13,7 +13,8 @@ class Command(BaseCommand):
     Loads necessary fixtures
     """
 
-    fixtures = ["base_schedule.json", "departments.json", "teacher.json"]
+    fixtures = ["base_schedule.json", "departments.json"
+                ]
```
Comment (NullDefault) [inline database/management/commands/load_fixtures.py]: why was this changed? seems unintentional

File: documentation/urls.py [hunk 2]
```diff
@@ -4,9 +4,11 @@
 from documentation import views as page_views
 
 urlpatterns = [
-    path('home-documentation', page_views.HomeDocView.as_view(), name='home-documentation'),
+    path('schedule-documentation', page_views.scheduleDocView.as_view(), name='schedule-documentation'),
     path('database-documentation', page_views.DBDocView.as_view(), name='database-documentation'),
-    path('admin-documentation', page_views.AdminDocView.as_view(), name='admin-documentation'),
+    path('datasheet-documentation', page_views.datasheetDocView.as_view(), name='datasheet-documentation'),
```
Comment (NullDefault) [inline documentation/urls.py]: incorrect naming scheme. All classes must be written AsClassNamesAre

File: documentation/views.py [hunk 3]
```diff
@@ -7,17 +7,19 @@
 from datacollection.models import PreferenceForm
 
 
-class HomeDocView(LoginRequiredMixin, TemplateView):
-    template_name = 'documentation/home-documentation.html'
-
+class scheduleDocView(LoginRequiredMixin, TemplateView):
```
Comment (NullDefault) [inline documentation/views.py]: naming scheme

File: templates/components/footer.html [hunk 4]
```diff
@@ -1,30 +1,35 @@
...
+                <h5 class="mb-6 font-bold uppercase">Help</h5>
+                <ul class="mb-4">
+                    <li class="mt-1">
+                        <a href="/" class="text-gray-800 hover:text-gray-100">Home</a>
+                    </li>
+                    <li class="mt-1">
+                        <a href="{% url 'pages:docs' %}" class="text-gray-800 hover:text-gray-100">Docs</a>
+                    </li>
+                    <li class="mt-1">
+                        <a href="/" onclick="doClick(); return false;"
```
Comment (NullDefault) [inline templates/components/footer.html]: this doesn't do anything

File: templates/components/footer.html [hunk 5]
```diff
@@ -34,3 +39,6 @@ <h5 class="mt-4 text-gray-800">© 2021 by DrewUniversityCS</h5>
     <div class="h-24 bg-foundation w-full">
     </div>
 </footer>
+
+<script src = "{% static 'js/searchDocs.js' %}">
```
Comment (NullDefault) [inline templates/components/footer.html:40]: there is no file at the specified location - did you not add it?

File: templates/documentation/datasheet-documentation.html [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+                                            corresponding file to its category
+                                        </li>
+                                    </ul>
+                                    <div class="cursor-default"
+                                         @click="selected !== 2 ? selected = 2 : selected = null">
+                                        <span>
+                                            [
+                                        </span>
+                                        <span class="hover:underline text-blue-500">
+                                            close
```
Comment (NullDefault) [inline templates/documentation/datasheet-documentation.html]: why this weird close button for everything? An icon or something would look much nicer

File: templates/documentation/datasheet-documentation.html [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+                            </div>
+                        </button>
+                        <div id = "select4" class="relative overflow-hidden transition-all max-h-0 duration-700" style=""
+                             x-ref="container4"
+                             x-bind:style="selected == 4 ? 'max-height: ' + $refs.container4.scrollHeight + 'px' : ''">
+
+                            <div class="text-left items-center">
+                                <div class="p-6">
+                                    <ul class="list-disc pl-8">
+                                        <img src="{% static 'images/ds4.gif' %}"
```
Comment (NullDefault) [inline templates/documentation/datasheet-documentation.html]: this image is barely visible - the resolution is super low. Also why is this a gif? Isn't it a static image?

Comment (NullDefault) [reply templates/documentation/datasheet-documentation.html]: In general many of the images are super low resolution

Comment (NullDefault) [reply templates/documentation/datasheet-documentation.html]: Yeah looking through the pages, the screenshots are too low resolution for them to communicate the information they carry

File: templates/documentation/datasheet-documentation.html [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+                    <!--Title-->
+                    <div>
+                        <h1 class="break-normal text-black pt-6 pb-2 text-5xl">Overview</h1>
+                        <hr class="border-b border-grey-light">
+                    </div>
+                    <p class="py-6">
+                        Admin will be able to upload CSV files pertaining to 3 categories: Courses, Students, or
+                        Preferences. Uploading a preset CSV will prevent admin from having to manually input respective
+                        data one by one. If Admin does not have any CSV files to upload, please refer to the Database
+                        documentation for next steps [<a class="hover:underline text-blue-500" href="{% url 'documentation:database-documentation' %}">here</a>]
```
Comment (NullDefault) [inline templates/documentation/datasheet-documentation.html]: The "here" link does not work on some of the pages

Comment (NullDefault) [reply templates/documentation/datasheet-documentation.html]: Also, some of the doc links don't work

**Review by NullDefault — CHANGES_REQUESTED** [review]
Major issues:

- The images are low resolution and in the gif format despite not being animated. Switching over to an external data store for the images is recommended.
- Many of the links on the pages do not work, need fixing.
- The "help me" button does not work - I suspect there was an issue with git.
- The text should be styled more consistently with the rest of the application. The only glaring issue is how huge some of the text is.

File: database/management/commands/load_fixtures.py (hunk 1 above)
Comment (abax17) [reply database/management/commands/load_fixtures.py]: must have been a mistake my apologies

File: documentation/urls.py (hunk 2 above)
Comment (abax17) [reply documentation/urls.py]: will change

File: documentation/views.py (hunk 3 above)
Comment (abax17) [reply documentation/views.py]: yup will fix this

File: templates/components/footer.html (hunk 4 above)
Comment (abax17) [reply templates/components/footer.html]: this will be fixed when i push the js

File: templates/documentation/datasheet-documentation.html (hunk 6 above)
Comment (abax17) [reply templates/documentation/datasheet-documentation.html]: sure will add this

File: templates/documentation/datasheet-documentation.html (hunk 8 above)
Comment (abax17) [reply templates/documentation/datasheet-documentation.html]: some of the docs where not commited. will commit them with the js

File: static/js/searchDocs.js [hunk 9]
```diff
@@ -0,0 +1,39 @@
+function doClick() {
+       if(String(document.location.href).includes("/dataingest/upload")){
```
Comment (NullDefault) [inline static/js/searchDocs.js]: You are translating the href into a string each time you do a comparison. Just do it once and put it into a variable

File: static/js/searchDocs.js [hunk 10]
```diff
@@ -0,0 +1,39 @@
...
+         document.location.href = "../../datasheet-documentation";
```
Comment (NullDefault) [inline static/js/searchDocs.js]: You could also probably do this with a dictionary object in way fewer lines, but that's not essential (although much cleaner)

File: theme/static/css/styles.css.map [hunk 11]
```diff
@@ -0,0 +1 @@
...S;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAAS;;EAAT;IAAA;EAA
```
Comment (NullDefault) [inline theme/static/css/styles.css.map]: rip you pushed up the un-built version of tailwind I think

**Review by NullDefault — CHANGES_REQUESTED** [review]
